mgmt_cli_test.py

```diff
@@ -275,6 +275,18 @@ class MgmtCliTest(BackupFunctionsMixIn, ClusterTester):
     LOCALSTRATEGY_KEYSPACE_NAME = "localstrategy_keyspace"
     NETWORKSTRATEGY_KEYSPACE_NAME = "networkstrategy_keyspace"
 
+    def _get_cluster(self, manager_tool, force_add: bool = False):
```

Comment:
`_get_cluster` which is adding cluster... sounds misleading.
how about `_ensure_and_get_cluster`?

Reply:
agree, make sense, will fix it

Reply:
Fixed, testing here:
https://jenkins.scylladb.com/job/scylla-staging/job/mikita/job/sct/job/manager-master/job/ubuntu22-sanity-test/10/